docs/csharp/tuples.md

```diff
@@ -26,24 +26,26 @@ different ways to use them, and initial guidance on working with Tuples.
 
 Let's start with the reasons for adding new Tuple support. Methods return
 a single object. Tuples enable you to package multiple values in that single
-object more easily.
+object more easily. You don't need behavior on these types, but you want to
```

Comment:
Understanding the sentence requires that you know that tuples don't allow you to define custom methods. I'm not sure how many readers will have the necessary context. I think a separate paragraph about the consequences of being lightweight might be better here.